pkg/synopsysctl/alert_migrate.go

```diff
@@ -166,7 +166,13 @@ func migrateAlert(alert *v1.Alert, operatorNamespace string, flags *pflag.FlagSe
 	if svc != nil {
 		svc.Kind = "Service"
 		svc.APIVersion = "v1"
+		if svc.Labels == nil {
+			svc.Labels = make(map[string]string)
+		}
 		svc.Labels["name"] = newReleaseName
+		if svc.Spec.Selector == nil {
```

Comment:
pls do the above comment change here as well

Reply:
done!